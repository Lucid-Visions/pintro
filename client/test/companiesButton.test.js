import React from 'react';
import renderer from 'react-test-renderer';
import CompaniesButtonComponent from '../src/components/CompaniesButton';

test('Companies button renders correctly', () => {
    const component = renderer.create(<CompaniesButtonComponent />).toJSON();
    expect(component).toMatchSnapshot();
});