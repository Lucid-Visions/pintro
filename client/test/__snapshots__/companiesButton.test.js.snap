// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Companies button renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "shadowOffset": Object {
          "height": 6,
        },
        "shadowOpacity": 0.195,
        "shadowRadius": 5.4,
      },
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderRadius": 15,
        "flexDirection": "row",
        "padding": 20,
        "resizeMode": "contain",
        "width": "85%",
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderColor": "red",
        "flex": 1,
        "flexDirection": "row",
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "center",
          "marginLeft": 0,
          "padding": 2,
        }
      }
    >
      <Image
        source={1}
        style={
          Object {
            "borderRadius": 75,
            "height": 40,
            "overflow": "hidden",
            "width": 40,
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "flexDirection": "column",
          "left": "auto",
          "marginLeft": 5,
          "marginTop": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 16,
            "fontWeight": "bold",
          }
        }
      />
      <Text
        style={
          Object {
            "color": "grey",
            "fontSize": 12,
            "fontWeight": "normal",
          }
        }
      />
    </View>
  </View>
</View>
`;
